pkgs/development/python-modules/geographiclib/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, isPy27
+}:
+
+buildPythonPackage rec {
+  pname = "geographiclib";
+  version = "1.49";
+  disabled = false;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "635da648fce80a57b81b28875d103dacf7deb12a3f5f7387ba7d39c51e096533";
+  };
+
+  doCheck = false;  # too much
```

Comment:
Too much what? Please add a meaningful comment

Reply:
This was copy-pasted from geopy package. It probably shouldn't belong in this newly created file.